### PR TITLE
add steps to metrics page

### PR DESCRIPTION
**Pull request by whatrocks**
* add details about steps to training metrics page

### CODE REVIEW

**Review by ReDeiPirati — APPROVED** [review]

File: docs/guides/jobs/metrics.md [hunk 1]
```diff
@@ -52,7 +52,25 @@ print('{{"metric": "loss", "value": {}}}'.format(loss))
...
+### Steps
+
+By default, Training Metrics will show `Time` as the x-axis. However, you can also toggle the Training Metrics to display steps, as long as you provide that information in your logs.
+
+Here is the basic format. Notice that you'll need to provide an integer value for the `step` field:
+```python
+print('{"metric": "<metric_name>", "value": <int_or_float>, "step": <int>}')
+```
+
+We will also display `epoch` as a Step if you emit this value. Here's an example:
```
Comment (narenst) [inline docs/guides/jobs/metrics.md]: This seems like a format and not an example. Can you also include examples (similar to the previous section):

```
print('{"metric": "accuracy", "value": 0.985, "epoch": 3}')
# {"metric": "accuracy", "value": 0.985. "epoch": 3}
```

**Review by narenst — APPROVED** [review]

**Comment by ReDeiPirati** [comment]
Question: should it be helpful to have code example for Keras (callbacks), TensorFlow (hooks) and PyTorch?

**Comment by whatrocks** [comment]
> Question: should it be helpful to have code example for Keras (callbacks), TensorFlow (hooks) and PyTorch?

Definitely! I'd love if you could add some examples after I publish this

File: docs/guides/jobs/metrics.md (hunk 1 above)
Comment (whatrocks) [reply docs/guides/jobs/metrics.md]: Yes!
